### ISSUE TITLE
Auto-connect

### ISSUE THREAD

**Issue by Simounet** [hakobe/homura#4]
Hi,
I just want to know why you aren't connecting bouncers to their server at homura's starting ? Is there a reason to wait for a user ?

**Comment by hakobe** [comment]
No, there are no obvious reasons that homura does not start connecting at startup. 

When I wrote this code, I thought that it is simple and clear to call `bouncer.connect()` only at one place, but now I feel that it is nice idea to connect to server on homura's startup. ( My irc logs might be lost by this behavior...)
